client/app/dbaas/logs/detail/streams/home/logs-streams-home.html

```diff
@@ -1,4 +1,40 @@
-<div>
-    Streams (home)
-    Service Name: {{ ctrl.serviceName }}
-</div>
+
+<section class="cui-page__content">
+    <cui-page-content-title text="'logs_streams_title' | translate"></cui-page-content-title>
+    <p data-translate="logs_streams_description"></p><br />
```

Comment:
No br tag should be needed. Especially with p tags ;)

Reply:
removed